docs/static/openai-spec.yml

```diff
@@ -0,0 +1,2725 @@
+openapi: 3.1.0
+info:
+  title: OpenAI API
+  description: The OpenAI REST API. Please see https://platform.openai.com/docs/api-reference for more details.
+  version: 2.3.0
```

Comment:
I'm wondering if we should either:

* put this into a 2.3.0 directory
* or add -2.3.0 to the file name


Or perhaps, are we always replacing / overwritting with a new spec?